package.json

```diff
@@ -1165,6 +1177,18 @@
                     "category": "Notebook",
                     "when": "notebookEditorFocused"
                 },
+                {
```

Comment:
when = false in the commandPalette placement section to hide from command palette. These commands are just for keybindings.